src/Lachain.Core/Consensus/BlockProducer.cs

```diff
@@ -64,14 +66,15 @@ ITransactionBuilder transactionBuilder
             _blockManager = blockManager;
             _stateManager = stateManager;
             _transactionBuilder = transactionBuilder;
+            txPerBlock = Convert.ToInt32(configManager.GetConfig<BlockchainConfig>("blockchain")?.TargetTransactionsPerBlock ?? 1_000);
```

Comment:
I think it is better to declare TargetTransactionsPerBlock as int in BlockchainConfig instead of this conversion

Reply:
okay. updating.